packages/fxa-support-panel/.gitignore

```diff
@@ -0,0 +1,6 @@
+/coverage.html
+/node_modules
+/dist
+*.gpg
+*.log
+npm-debug.log
```

Comment:
do you need npm-debug.log, if you already have *.log?

Reply:
Probably not, but that is an artifact from the other packages that was copypasta'd in here too it seems. We should prolly revisit these in all the sub-packages.

Reply:
that's what i thought!